### PR TITLE
update for skopeo util

### PR DESCRIPTION
**Pull request by spfrancis**
## what
* Skopeo util will be used for testing registry access to gitlab.

## why
* Skopeo utility will be used to augment UAT testing for gitlab.
* 

## references
* Link to any supporting github issues or helpful documentation to add some context (e.g. stackoverflow). 
* Use `closes #123`, if this PR closes a GitHub issue `#123`

### CODE REVIEW

File: src/docker/Dockerfile [hunk 1]
```diff
@@ -136,6 +136,13 @@ RUN dnf -y install podman-${PODMAN_VERSION} \
...
   && rm -rf /var/tmp/* /tmp/* /var/tmp/.???* /tmp/.???*
 
+# Install Skopeo. Get available versions by running the command `dnf list skopeo`
+ARG SKOPEO_VERSION=“1.2.0”
+ENV SKOPEO_VERSION=${SKOPEO_VERSION}
+RUN dnf -y install skopeo-${SKOPEO_VERSION} \
+  && dnf clean all \
+  && rm -rf /var/cache/yum/ \
+  && rm -rf /var/tmp/* /tmp/* /var/tmp/.???* /tmp/.???*
 RUN useradd -ms /bin/bash anvil
```
Comment (RothAndrew) [inline src/docker/Dockerfile:147]: add a blank line to separate this from the new code block

**Review by RothAndrew — CHANGES_REQUESTED** [review]
Missing new automated test

File: src/docker/Dockerfile (hunk 1 above)
Comment (spfrancis) [reply src/docker/Dockerfile:147]: applied those two changes.

**Review by RothAndrew — DISMISSED** [review]

**Review by spfrancis — COMMENTED** [review]
update in spec file

**Comment by spfrancis** [comment]
update with spec file and extra lines in dockerfile

**Review by RothAndrew — DISMISSED** [review]

**Review by saic-statonb — DISMISSED** [review]
These look good to me.

**Review by RothAndrew — DISMISSED** [review]

**Review by spfrancis — COMMENTED** [review]
looks good

File: src/docker/Dockerfile [hunk 2]
```diff
@@ -136,6 +136,14 @@ RUN dnf -y install podman-${PODMAN_VERSION} \
   && rm -rf /var/cache/yum/ \
   && rm -rf /var/tmp/* /tmp/* /var/tmp/.???* /tmp/.???*
 
+# Install Skopeo. Get available versions by running the command `dnf list skopeo`
+ARG SKOPEO_VERSION=“1.2.0”
```
Comment (RothAndrew) [inline src/docker/Dockerfile]: Change smart quotes to regular double quotes

**Review by RothAndrew — CHANGES_REQUESTED** [review]

File: src/docker/Dockerfile (hunk 2 above)
Comment (spfrancis) [reply src/docker/Dockerfile]: done

**Review by RothAndrew — APPROVED** [review]

**Pull request merged**